package.json

```diff
@@ -76,7 +76,7 @@
     "eslint-plugin-simple-import-sort": "^7.0.0",
     "eslint-plugin-unused-imports": "^1.1.2",
     "jest": "^27.0.6",
-    "typescript": "~4.0.0"
+    "typescript": "^4.0.0"
```

Comment:
Thanks for co-authoring @filiptronicek ❤️

Reply:
Sure thing <3